jujugui/static/gui/src/app/components/deployment/summary/summary.js

```diff
@@ -43,6 +43,55 @@ YUI.add('deployment-summary', function() {
       validateForm: React.PropTypes.func.isRequired
     },
 
+    getInitialState: function() {
+      return {
+        regions: [],
+        activeRegion: this.props.deploymentStorage.region
```

Comment:
I don't think that's the case but I'll check in qa just to be sure. If anything 'activeRegion' will simply be undefined